tests/python/test_python_frontend.py

```diff
@@ -2446,7 +2445,7 @@ def fusion_func(
             fd.add_output(t_out)
 
         in_tensors = [inputs_mk_nk, inputs_mk_kn, inputs_km_nk, inputs_km_kn]
-        use_bias = [None, bias0d, bias1d, bias2d]
+        use_bias = [None, bias0d, bias1d]
```

Comment:
Does 2D bias not work or are we just not going to support it since it's not mentioned in the `F.linear` docs?

Reply:
Oh I guess this is because we cannot map domains properly in that case? https://github.com/NVIDIA/Fuser/pull/2240#discussion_r1600726831

I wonder if this is a case we will encounter. We should be sure to guard our thunder prim so that it does not send us linears with 2D bias.

Reply:
https://github.com/Lightning-AI/lightning-thunder/blob/2fa97d6b1fead1eb9d0dc17703ebb12bbf752bbc/thunder/core/prims.py#L3456. They are not supported by Thunder either.